### ISSUE TITLE
Release notes languages management 

### ISSUE THREAD

**Issue by ChumachenkoRoman** [Triple-T/gradle-play-publisher#1104]
Our app currently is published in different regions but uses only English release notes. Is there any way to publish single release notes file for all languages?

**Comment by SUPERCILEX** [comment]
You'd have to implement this, but yeah probably.

**Comment by ChumachenkoRoman** [comment]
We used straightforward approach: we created default files for all available languages and created simple bash script which writes same release notes to each one. So we update all those files before assembling release build and that's it. 

I was curious if there is another way to achieve same result?

**Comment by SUPERCILEX** [comment]
Yeah, no better way unfortunately. This is a very reasonable request though so I'd accept a PR that supports a default language folder. The one thing that concerns me is how we upload all those languages to the Play API: do we need to hardcode a list of language codes? Ideally the Play API would support a default.

**Comment by ChumachenkoRoman** [comment]
Maybe I'm getting you wrong here but the default file should be located under a language-named directory for it to be handled to play correctly

**Comment by SUPERCILEX** [comment]
Yeah that's correct. I'm saying that if someone wants to open a PR to change that, I'm onboard.
